src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -71,12 +72,7 @@ GUID GlobalAppSettings::GetDefaultProfile() const noexcept
 
 AppKeyBindings GlobalAppSettings::GetKeybindings() const noexcept
 {
-    return _keybindings;
-}
-
-void GlobalAppSettings::SetKeybindings(winrt::TerminalApp::AppKeyBindings newBindings) noexcept
-{
-    _keybindings = newBindings;
+    return *_keybindings;

```

Comment:
Does this make a copy? Should it just be a ref?

Reply:
This does winrt magic - winrt will take the implementation::AKB, and turn that into a AKB, which is a magic ref-counted winrt object. 
> An implicit conversion provides a (very efficient) conversion from the implementation type to the projected type, and that's very convenient for many scenarios. Without that facility, a lot of implementation types would prove very cumbersome to author. 
[source](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/diag-direct-alloc)

Reply:
Ahhhhh ok.